### DISCUSSION TITLE
Discharge Time of LTO Battery not right, even when defining current and capacity

### DISCUSSION THREAD

**Discussion by bogicaa** [pybamm-team/PyBaMM#4062] [Q&A]
Hello,

Recently I found a way to improve the shape of my voltage curve for LTO battery. I defined all the parameters except initial concentrations in the electrodes. When trying to calculate these in Comsol and then using them in simulation, it got completely different results. Therefore I tried to use Pybamms initial_soc option when running a simulation to see if it will fit the curve the right way and then compare it with the measured OCV. I tried doing this with 2 A current for a  20 Ah battery, meaning it should have been discharged after 10 hours, which was not the case… I am not sure why this happened and how could I influence this? 

As said, now I am satisfied with the shape of the curve, but the discharge time is not right…When putting the simulation time to 36000 seconds, it only plots the part of the curve, when using 72000(20 hours) I can see a whole curve but it takes 15 hours to discharge the battery instead of 10…

Looking forward to your response,

Ogi

**Comment by valentinsulzer** [comment]
The main parameters that affect this are electrode height, width, and thicknesses + maximum concentration + active material volume fraction. Make sure those are the same as in comsol. 

You can use the electrode state of health model https://docs.pybamm.org/en/latest/source/examples/notebooks/models/electrode-state-of-health.html to check the OCV behavior of your battery
